apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/PartitionIdPartial.cs

```diff
@@ -23,13 +23,25 @@ public partial class PartitionId
         /// </summary>
         /// <param name="projectId">The project ID of the partition. Must not be null.</param>
         /// <param name="namespaceId">The namespace ID of the partition. Must not be null.</param>
-        /// <param name="databaseId">The database ID of the partition. Must not be null.</param>
-        public PartitionId(string projectId, string namespaceId = "", string databaseId = "") : this()
+        public PartitionId(string projectId, string namespaceId = "") : this()
```

Comment:
Any reason not to do what you did elsewhere, and just call `this(projectId, namespaceId, "")` with an empty body?

Reply:
It was a miss, will rectify in next upload.